tests/src/test/resources/pants.conf

```diff
@@ -3,12 +3,27 @@ include required("versions.conf")
 probe {
   driver {
     launch.timeout = "2 minutes"
-    headless = false # make all pants test run in ui mode by default for more accurate results
-    check.freezes = true # fail test in case of detecting freeze
+    vmOptions = [
+      "-Dvcs.log.index.git=false",
+      "-Dgit.process.ignored=false",
+      "-Ddotty.highlighting.compiler.errors.in.editor=false",
+      "-Dvcs.root.detector.folder.depth=1",
+      "-Xms2g",
+      "-Xmx12g",
```

Comment:
The standard runner of Github Actions doesn't [have](https://docs.github.com/en/free-pro-team@latest/actions/reference/specifications-for-github-hosted-runners#supported-runners-and-hardware-resources) that much RAM.

Reply:
I just copied settings that we use. So let's roll that back.